process/incubation.md

```diff
@@ -0,0 +1,138 @@
+# Swift Server Work Group Incubation Process
+
+version 1.0
+
+## Overview
+
+As described on https://swift.org/server/, the goal of the Swift Server Work Group (SSWG) is to create a robust, healthy ecosystem for server application development with Swift. One avenue to achieve this goal is to encourage the development of high quality, well maintained libraries and tools that the community can comfortably lean on.
+
+The difference between the SSWG and the Swift Evolution process is that server-oriented libraries and tools that are produced as a result of work group efforts will exist outside of the Swift language project itself, and they will be distributed across different code bases.
+
+The teams at Apple, IBM, and Vapor have engineers that will actively participate in the development of such libraries and tools, and we would love to see the community joining in this effort. To that end, the work group defined and launches an incubation process where anyone can pitch, propose, develop, and contribute to such libraries and tools.
+
+The incubation process is designed to help nurture and mature projects ensuring standardization, quality, and longevity. It also seeks to increase the visibility of ideas, experiments, or other early work that can add value to the SSWG mission. The following document details this incubation process.
+
+## Process
+
+Incubation is made of the following stages: **Pitch**, **Proposal**, **Development**, and **Recommendation**. The Development stage is where the majority of incubation take place. The SSWG will maintain a public "Swift Server Ecosystem" index page that will list all recommended tools and libraries as well as projects that are part of the incubation process and their respective incubation level.
+
+### Pitch
+
+Pitches are written proposals of new libraries or tools including ideas for new features in existing libraries or tools. They are used to collect feedback from the community and help define the exact scope of a project prior to writing code. Pitches are submitted by creating a new thread in the Swift Server forum area.
+
+### Proposal
+
+For a pitch to be moved into the Proposal stage, it must be endorsed by at least two members of the SSWG. The scope of the proposed code needs to closely align with the endorsed Pitch and it is subject to review based on the SSWG graduation criteria defined below. Proposal are submitted by creating a new thread in the Swift Server forum area with a link to the code repository in question.  Proposals submitted to the SSWG must provide the following information:
+* Name (must be unique within SSWG)
```

Comment:
Since the format for a proposal is so structured, we could follow the same model a Evolution proposals and have a "proposals" folder on this repository.  I'm fine with creating threads as well, although I'd recommend they go into a subcategory "Development > Server > Proposals" so they stay organized.  The top (pinned) post could articulate what's required in a proposal.

Reply:
@tkremenek i was thinking having both: 
=> a "proposals" folder on this repository (placeholder exists)
=> a subcategory in the forums where we also paste an "elevator pitch" and the link to the proposal

if too much we can just do the first

Reply:
That sounds reasonable to me.  Let's try and avoid too much duplication and friction where possible, but having the proposal folder in the repository seems reasonable and has precedent.

Are proposals going to be numbered, similar to how they are with Swift Evolution?